unbound_script.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 the v8go contributors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package v8go
+
+// #include <stdlib.h>
+// #include "v8go.h"
+import "C"
+import "unsafe"
+
+type UnboundScript struct {
+	ptr C.UnboundScriptPtr
+}
+
+// NOTE: This can only be run in a context that belongs to the isolate
+// that this unbound script was compiled in
+func (u *UnboundScript) Run(ctx *Context) (*Value, error) {
+	// TODO: Validate that they are running the unbound script in a context that belongs to the same isolate
+	rtn := C.UnboundScriptRun(ctx.ptr, u.ptr)
+	return valueResult(ctx, rtn)
+}
+
+type CachedData struct {
+	ptr      *C.CachedData
+	data     []byte
+	length   int
+	rejected int
```

Comment:
This will need to be created in another process from a byte slice.  Since public attributes are used in the V8 API, we could probably just make the fields exported Go fields so it could similarly be created directly.

```suggestion
	Data     []byte
	Rejected bool
```